src/14_cal.py

```diff
@@ -21,4 +21,21 @@
 
 import sys
 import calendar
-from datetime import datetime
\ No newline at end of file
+from datetime import datetime
+
+user_input = input('type month and year ')
```

Comment:
you might want to add more input to the user to help them understand what they need to put in, or what they can expect from the program if they do not put in certain default values. still, great work!